articles/basics/installation/atom-plugin.md

```diff
@@ -21,7 +21,7 @@ From there, search for the `fuse` package and click Install, as shown below.
 ## From the command line
 
 Run the following command:
-
+```s
```

Comment:
The correct language is `sh` (or `shell`). #69 adds support for this.

* All `s` in this PR can be replaced with `sh`.